independent-publisher-connectors/Finnish BIS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,716 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Finnish BIS",
+    "description": "Get data from the Finnish Business Information System (BIS) maintained by the PRH and the Finnish Tax Administration. \n\nIn addition to basic details of businesses or companies registered at the Finnish Trade Register, the service also provides information on other registers they have been added to: the Prepayment Register, the Employer Register, the VAT Register, the register of bodies liable for tax on insurance premiums, and the Client Register of the Tax Administration.",
+    "version": "1.0",
+    "contact": {
+      "name": "Timo Pertilä",
+      "url": "https://timopertila.com",
+      "email": "timo.pertila@gmail.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://tietopalvelu.ytj.fi/yrityshaku.aspx?kielikoodi=3"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.ytj.fi/en/index/filedescriptions.html"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Management"
+    }
+  ],
+  "host": "avoindata.prh.fi",
+  "basePath": "/",
+  "schemes": [
+    "http"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/bis/v1/{businessId}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "type": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Type"
+                },
+                "version": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Version"
+                },
+                "totalResults": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "Total results"
+                },
+                "resultsFrom": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "resultsFrom",
+                  "title": "Results form"
+                },
+                "previousResultsUri": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Previous results URL"
+                },
+                "nextResultsUri": {
+                  "type": "string",
+                  "description": "nextResultsUri",
+                  "title": "Next results URL"
+                },
+                "exceptionNoticeUri": {
+                  "type": "string",
+                  "description": "",
```

Comment:
@tpertila Empty property values can be removed until unless you have a specific reason to keep them.